test/e2e/service.go

```diff
@@ -37,14 +37,36 @@ import (
 var _ = Describe("Services", func() {
 	var c *client.Client
 	// Use these in tests.  They're unique for each test to prevent name collisions.
+	var ns_ *api.Namespace
 	var namespace0, namespace1 string
+	var err error
 
 	BeforeEach(func() {
-		var err error
+
 		c, err = loadClient()
 		Expect(err).NotTo(HaveOccurred())
-		namespace0 = "e2e-ns-" + dateStamp() + "-0"
-		namespace1 = "e2e-ns-" + dateStamp() + "-1"
+		namespace0 = "e2e-ns-0"
+		namespace1 = "e2e-ns-1"
+
+		ns_, err = createTestingNS(namespace0, c)
```

Comment:
ns_ is only used within this scope, so please remove it from the outer scope.  And please rename it to something without a trailing underscore, as we don't use that anywhere else.  Plain 'ns' should be fine.